### PR TITLE
Landing-Page updates

### PR DESCRIPTION
**Pull request by jenbeckett**
# Description

This PR fleshes out the Landing-Page.vue file. This page will dynamically render based on if the user's application is one of these states: Not Started, Draft, In Progress (submitted) and Completed. 

The states are currently located in backend/src/components/user.js. At a later time when the API is complete, this will be filled from a GET request. 

TODO: currently the "search by facility" is just a textbox. 

## Types of changes

 New feature (non-breaking change which adds functionality) 

## Checklist

- [x ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: backend/src/components/user.js [hunk 1]
```diff
@@ -9,11 +9,24 @@ async function getUserInfo(req, res) {
...
+      applicationSubmitted: true,
+      applicationApproved:
+  */
+
   let resData = {
     displayName: `${req.session.passport.user.displayName}`,
     organizationList: [{
       organizationName: 'ABC organization',
       organizationId: 'org123',
+      isDraft: false,
```
Comment (roblo-cgi) [inline backend/src/components/user.js:26]: can we consolidate isDraft, applicationSubmitted, applicatinApproved to be one status code with DRAFT, SUBMITTED, APPROVED

File: frontend-full-build/package-lock.json [hunk 2]
```diff
@@ -20597,6 +20597,12 @@
         "decamelize": "^1.2.0"
       }
     },
+    "node_modules/jquery": {
```
Comment (roblo-cgi) [inline frontend-full-build/package-lock.json:20600]: please double check package-lock.json file.  Did we change the package.json

**Review by roblo-cgi — CHANGES_REQUESTED** [review]

File: frontend-full-build/package-lock.json (hunk 2 above)
Comment (jenbeckett) [reply frontend-full-build/package-lock.json]: resolved- my package.lock should match main now.

File: backend/src/components/user.js (hunk 1 above)
Comment (jenbeckett) [reply backend/src/components/user.js]: done!

**Pull request closed without merging**